src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        alt="Photo of product"
+        class="product-card__image"
+      />
+
+      <h2 class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__code">Product code: 195434</p>
+
+      <section class="product-card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product-card__reviews">Reviews: 5</p>
+      </section>
+
+      <section class="product-card-price">
+        <p class="product-card-price__label">Price:</p>
+        <p class="product-card-price__value">$2,199</p>
+      </section>
+
+      <button
+        href="#"
```

Comment:
The 'button' element should not have an 'href' attribute. If you want to make the button act like a link, consider using an 'a' tag or handling the click event with JavaScript.